pyartcd/pyartcd/pipelines/prepare_release.py

```diff
@@ -650,24 +650,13 @@ async def verify_attached_operators(self, *advisories: List[int]):
         await exectools.cmd_assert_async(cmd, env=self._elliott_env_vars, cwd=self.working_dir)
 
     @retry(reraise=True, stop=stop_after_attempt(3), wait=wait_fixed(10))
-    def send_live_id_request_mail(self, advisories: Dict[str, int]):
-        subject = f"Live IDs for {self.release_name}"
-        main_advisory = "image" if self.release_version[0] >= 4 else "rpm"
-        content = f"""Hello docs team,
-
-ART would like to request Live IDs for our {self.release_name} advisories:
-{main_advisory}: https://errata.devel.redhat.com/advisory/{advisories[main_advisory]}
-
-This is the current set of advisories we intend to ship:
-"""
+    async def create_live_ids(self, advisories: Dict[str, int]):
         for kind, advisory in advisories.items():
-            content += (
-                f"- {kind}: https://errata.devel.redhat.com/advisory/{advisory}\n"
+            _LOGGER.info(f"Giving {kind} advisory {advisory} a Live ID.")
+            await exectools.cmd_assert_async(
+                f"curl -X POST --negotiate -u : https://errata.devel.redhat.com/errata/set_live_advisory_name/{advisory}\n"
```

Comment:
Would like to add `--fail` option to return non-zero code if response status is not 200-399.

Reply:
did that in [elliott](https://github.com/openshift/elliott/commit/9e4a6b3efb7eda5c9c4734d610d5644ca64cd298)